tests/unit/migrations/namespace-form-fields.js

```diff
@@ -4,6 +4,37 @@ var sinon = require('sinon'),
     utils = require('../utils'),
     migration = require('../../../migrations/namespace-form-fields');
 
+// Arguments : docArrayForFirstBatch, docArrayForSecondBatch, ...
+var makeStubs = (...viewBatches) => {
+  console.log(viewBatches);
+  var getView = sinon.stub(db.medic, 'view');
+  if (viewBatches.length === 0) {
+    getView.callsArgWith(3, null,
+      {
+        total_rows: 0,
+        rows: []
+      });
+  } else {
+    var totalRows = viewBatches.reduce(function(total, batch) {
```

Comment:
FWIW ES6: this can just be `viewBatches.reduce((total, batch) => total + batch.length, 0);`